_posts/2020-07-07-Solidity-0611-release-announcement.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.6.11 Release Announcement'
+date: '2020-07-07'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.6.11](https://github.com/ethereum/solidity/releases/tag/v0.6.11) adds inheritance to NatSpec comments,
+improves debugging data output and fixes some minor issues with opening up ``calldata`` for non-external functions.
+
+
+## Notable New Features 
+
+### NatSpec Inheritance and on Events
+
+NatSpec comments are a way to describe the behaviour of a function to end-users. It also allows
+to provide more detailed information to developers.
+
+One regular use-case is that you document the behaviour of an interface and then implement
+the interface in a derived contract. Previously, you had to repeat the documentation in the derived
+contract. This is now obsolete: The compiler will automatically inherit the documentation of
+the base function if the derived function does not provide any NatSpec tag.
+
+If you provide any of the tags (``@param``, ``@dev``, ...), then nothing will be inherited.
+The next release will provide a feature to explicitly inherit from a certain base also in that case,
+so stay tuned!
+
+
+Furhermore, NatSpec is now supported on events.
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.11;
+interface Gathering {
+  /// The address `participant` just registered for the gathering.
```

Comment:
This becomes a notice without an explicit `@notice` tag?

Reply:
@axic Yes. Notice is the "default."